pkg/command/server.go

```diff
@@ -160,25 +160,89 @@ func Server(cfg *config.Config) *cli.Command {
 					// TODO read users for the config backend from config file
 					Users: []glauthcfg.User{
 						glauthcfg.User{
-							Name:         "admin",
+							Name:         "einstein",
+							GivenName:    "Albert",
+							SN:           "Einstein",
 							UnixID:       20000,
 							PrimaryGroup: 30000,
-							Mail:         "admin@example.com",
-							PassSHA256:   "8c6976e5b5410415bde908bd4dee15dfb167a9c873fc4bb8a81f6f2ab448a918", // admin
+							OtherGroups:  []int{30001, 30002, 30007},
+							Mail:         "einstein@example.org",
+							PassSHA256:   "69bf3575281a970f46e37ecd28b79cfbee6a46e55c10dc91dd36a43410387ab8", // relativity
 						},
 						glauthcfg.User{
-							Name:         "demo",
+							Name:         "marie",
```

Comment:
@butonic shouldn't name be `curie` here to be aligned with the rest? food for thought